src/integrations/GA4/utils.js

```diff
@@ -228,24 +230,13 @@ function getPageViewProperty(props) {
  * @param {*} rudderElement
  * @param {*} measurementId
  */
-const proceedCloudMode = (rudderElement, measurementId) => {
-  const payload = rudderElement;
-  const cookieArr = document.cookie.split(";");
-  const cookieObj = {};
-  cookieArr.forEach((cookieEle) => {
-    const cookieElements = cookieEle.split("=");
-    const [first, second] = cookieElements;
-    cookieObj[first.trim()] = second;
-  });
-  const measurementIdArr = measurementId.split("-");
-  let sessionId;
-  if (cookieObj[`_ga_${measurementIdArr[1]}`]) {
-    sessionId = cookieObj[`_ga_${measurementIdArr[1]}`].split(".");
-    const GA4 = { sessionId: sessionId[2] };
-    payload.message.integrations = { All: true, GA4 };
-    return payload;
+const getGa4SessionId = (measurementId) => {
```

Comment:
@saikumarrs Any way to write unit tests within this PR for getGa4SessionId method ?

Reply:
as of now we are not supporting it, but all the scenarios are covered (even if we don't have sessionId then it will not brake)